README.md

```diff
@@ -1,109 +1,56 @@
-![Discord4J Logo](/images/d4j_l.png?raw=true)
+![Discord4J Logo](../images/d4j_l.png?raw=true)
 
-# Discord4J *v3* [![Download](https://jitpack.io/v/Discord4J/Discord4j.svg?style=flat-square)](https://jitpack.io/#Discord4J/Discord4j)  [![Support Server Invite](https://img.shields.io/badge/Join-Discord4J-7289DA.svg?style=flat-square)](https://discord.gg/NxGAeCY) [![Documentation Status](https://readthedocs.org/projects/discord4j/badge/?version=latest)](http://discord4j.readthedocs.io/en/latest/?badge=latest&style=flat-square)
+# Discord4J [![Maven Central](https://img.shields.io/maven-central/v/com.discord4j/Discord4J/2.svg?style=flat-square)](https://mvnrepository.com/artifact/com.discord4j/Discord4J)  [![JCenter](https://img.shields.io/bintray/v/austinv11/maven/Discord4J.svg?style=flat-square)](https://bintray.com/austinv11/maven/Discord4J/_latestVersion)  [![Support Server Invite](https://img.shields.io/badge/Join-Discord4J-7289DA.svg?style=flat-square&logo=discord)](https://discord.gg/NxGAeCY)
 
-A reactive Java interface for the official [Discord](https://discordapp.com/) API, written in Java 8. Discord4J v3 is designed using [reactive principles](https://www.reactivemanifesto.org/)
-using the [Reactor framework](https://projectreactor.io/) and built over [Netty](https://netty.io/) in order to create an efficient, fast and completely non-blocking API.
+A reactive Java wrapper for the REST and Gateway components of the official [Discord Bot API](https://discordapp.com/developers/docs/intro).
 
-For a quick overview on the benefits of reactive programming/Reactor as well as a quick tutorial on Reactor click here (TODO: add link!).
+Built with [Reactor](https://projectreactor.io/), [Netty](https://netty.io/), and a focus on flexibility, Discord4J provides an efficient, non-blocking interface for creating Discord bots.
```

Comment:
Tbh I like my tagline better. Because it mentions the java target version and links the reactive manifesto

Reply:
Im not sure the java target is relevant to be honest. 8 is assumed. Maybe we can work to combine them, I like how a single sentence sounds much more.

Reply:
8 is not assumed considering the rapid updates of java nowadays (and the next LTS release is right around the corner). I kinda think there should be a single sentence tagline (the first part) and then a more in-depth pitch as to why the user should use our library as opposed to others. So I think, if anything else, this section should actually be extended.

Reply:
I still think we may wanna rewrite this a bit, because I think a short tagline + longer explanation of the benefits is a good idea